Makefile

```diff
@@ -31,7 +31,7 @@ verify: ## Run verifications. Example: make verify
 .PHONY: verify
 
 build: ## Build the executable. Example: make build
-	go build -a -race -ldflags $(LDFLAGS) -o _output/hostpathplugin ./cmd/hostpathplugin
+	go build -a -mod=vendor -race -ldflags $(LDFLAGS) -o csi-driver-projected-resource ./cmd
```

Comment:
if you put it here, then we have to deal with our .gitignore and the new binary

whereas the _output dir is a convention we already have in place for openshift

Reply:
update pushed.